### ISSUE TITLE
Test fail with Pinto 0.09996 if there is a ~/.pause file

### ISSUE THREAD

**Issue by maxhq** [thaljef/Pinto#178]
I found out that for v0.09996 tests fail if there is an existing `~/.pause` file.

As soon as I remove the `~/.pause` file all tests except `t/02-bowels/42-install.t` succeed.
Message *Failed test 'Install a core module'* in `42-install.t` might have to do with my perlbrew environment.

The following tests are failing:
```
cpanm (App::cpanminus) 1.7024 on perl 5.020000 built for x86_64-linux
Work directory is /home/coje/.cpanm/work/1421363070.18010
You have make /usr/bin/make

...

    #   Failed test 'Distribution has correct dist path'
    #   at t/02-bowels/21-add-no-index.t line 25.
    #          got: 'M/MA/MAXHQ/Foo-Bar-0.01.tar.gz'
    #     expected: 'A/AU/AUTHOR/Foo-Bar-0.01.tar.gz'
    
    #   Failed test 'Path /tmp/BgEnYZ30DG/authors/id/A/AU/AUTHOR/CHECKSUMS should exist'
    #   at t/02-bowels/21-add-no-index.t line 25.
Can't call method "packages" on an undefined value at t/02-bowels/21-add-no-index.t line 28.
    # Child (Excluding with exact match) exited without calling finalize()

#   Failed test 'Excluding with exact match'
#   at /opt/perlbrew/perls/perl-5.20.0/lib/site_perl/5.20.0/Test/Builder.pm line 279.
# Tests were run but no plan was declared and done_testing() was not seen.
# Looks like your test exited with 2 just after 1.
t/02-bowels/21-add-no-index.t ......... 
Dubious, test returned 2 (wstat 512, 0x200)
Failed 1/1 subtests 

...
    
    #   Failed test 'Distribution has correct dist path'
    #   at t/lib/Pinto/Tester.pm line 218.
    #          got: 'M/MA/MAXHQ/DistA-1.tar.gz'
    #     expected: 'A/AU/AUTHOR/DistA-1.tar.gz'
    
    #   Failed test 'Path /tmp/WJ5Wax03qE/authors/id/A/AU/AUTHOR/CHECKSUMS should exist'
    #   at t/lib/Pinto/Tester.pm line 232.
    # Looks like you failed 2 tests of 9.

#   Failed test 'Skip all missing prereqs when adding'
#   at t/02-bowels/24-skip-prereqs.t line 26.
    
    #   Failed test 'Distribution has correct dist path'
    #   at t/lib/Pinto/Tester.pm line 218.
    #          got: 'M/MA/MAXHQ/DistA-1.tar.gz'
    #     expected: 'A/AU/AUTHOR/DistA-1.tar.gz'
    
    #   Failed test 'Path /tmp/WJ5Wax03qE/authors/id/A/AU/AUTHOR/CHECKSUMS should exist'
    #   at t/lib/Pinto/Tester.pm line 232.
    # Looks like you failed 2 tests of 12.

#   Failed test 'Skip all missing prereqs when pulling'
#   at t/02-bowels/24-skip-prereqs.t line 40.
    
    #   Failed test 'Distribution has correct dist path'
    #   at t/lib/Pinto/Tester.pm line 218.
    #          got: 'M/MA/MAXHQ/DistA-1.tar.gz'
    #     expected: 'A/AU/AUTHOR/DistA-1.tar.gz'
    
    #   Failed test 'Path /tmp/WJ5Wax03qE/authors/id/A/AU/AUTHOR/CHECKSUMS should exist'
    #   at t/lib/Pinto/Tester.pm line 232.
    # Looks like you failed 2 tests of 12.

#   Failed test 'Skip all named missing prereqs when pulling'
#   at t/02-bowels/24-skip-prereqs.t line 54.
# Looks like you failed 3 tests of 4.
t/02-bowels/24-skip-prereqs.t ......... 
Dubious, test returned 3 (wstat 768, 0x300)
Failed 3/4 subtests 

...

    # File [/tmp/SsAQZc0zLf/lib/perl5/strict.pm] does not exist
    
    # Looks like you failed 1 test of 5.

#   Failed test 'Install a core module'
#   at t/02-bowels/42-install.t line 145.
# Looks like you failed 1 test of 9.
t/02-bowels/42-install.t .............. 
Dubious, test returned 1 (wstat 256, 0x100)
Failed 1/9 subtests 

...

#   Failed test 'Result indicates action was succesful'
#   at t/02-bowels/73-stack-lock.t line 59.
# Log messages are...
# Target AUTHOR/Foo-2.tar.gz is not registered on stack master

#   Failed test 'Result indicates action was succesful'
#   at t/02-bowels/73-stack-lock.t line 60.
# Log messages are...
# Distribution AUTHOR/Foo-2.tar.gz is not in the repository
# Looks like you failed 2 tests of 24.
t/02-bowels/73-stack-lock.t ........... 
Dubious, test returned 2 (wstat 512, 0x200)
Failed 2/24 subtests 

...
```

**Comment by thaljef** [comment]
Thanks for the report. That looks like a very fixable bug. I'll see what I can do.

Have a great day!

**Comment by thaljef** [comment]
This was fixed and released as Pinto-0.09997.
